en/administration/centreon-ha/installation-4-nodes.md

```diff
@@ -0,0 +1,1019 @@
+
+```bash
+rm /etc/cron.d/centreon
+rm /etc/cron.d/centstorage
+rm /etc/cron.d/centreon-auto-disco
+```
+
+### Permission modifications
+
+Modifications have to be made on permissions of `/var/log/centreon-engine` and `/tmp/centreon-autodisco` directories.
+
+In a clustered-setup, it's a requirement to get a file sync and discovery scheduled task fully functionnal. 
+
+- Files synchronization
+
+```bash
+chmod 775 /var/log/centreon-engine/
+mkdir /var/log/centreon-engine/archives
+chown centreon-engine: /var/log/centreon-engine/archives
+chmod 775 /var/log/centreon-engine/archives/
+chmod 664 /var/log/centreon-engine/*
+chmod 664 /var/log/centreon-engine/archives/*
+```
+
+- Services discovery
+
+```bash
+mkdir /tmp/centreon-autodisco/
+chown apache: /tmp/centreon-autodisco/
+chmod 775 /tmp/centreon-autodisco/
+```
+
+### Stopping and disabling the services
+
+**Informations :** These operations must be applied to all nodes `@CENTRAL_MASTER_NAME@`, `@SLAVE_MASTER_NAME@`, `@DATABASE_MASTER_NAME@` et `@DATABASE_MASTER_NAME@`. All the Centreon suite is installed as a dependency of centreon-ha, but it will not be used on the database nodes and will not create any trouble.
+
+Centreon's application services won't be launched at boot time anymore, they will be managed by the clustering tools. These services must therefore be stopped and disabled:
+
+```bash
+systemctl stop centengine snmptrapd centreontrapd gorgoned cbd httpd24-httpd centreon mysql
+systemctl disable centengine snmptrapd centreontrapd gorgoned cbd httpd24-httpd centreon mysql
+```
+
+By default, the `mysql` service is enabled in both systemd and system V perspectives, so you'd rather make sure it is disabled:
+
+```bash
+chkconfig mysql off
+```
+
+### Creating the cluster
+
+#### Activating the clustering services
+
+First we enable all the services and start `pcsd` on both central nodes:
+
+```bash
+systemctl start pcsd
+```
+
+
+#### Preparing the server that will hold the function of *quorum device* 
+
+You can use one of your pollers to play this role. It must be prepared with the commands below: 
+
+```bash
+yum install pcs corosync-qnetd
+systemctl start pcsd.service
+systemctl enable pcsd.service
+pcs qdevice setup model net --enable --start
+pcs qdevice status net --full
+```
+
+Modify the parameter `COROSYNC_QNETD_OPTIONS` in the file `/etc/sysconfig/corosync-qnetd` to make sure the service will be listening the connections just on IPv4
+
+```bash
+COROSYNC_QNETD_OPTIONS="-4"
+```
+
+#### Authenticating to the cluster's members
+
+For the sake of simplicity, the `hacluster` user will be assigned the same password on both central nodes **and `@QDEVICE_NAME@`**.
+
+```bash
+passwd hacluster
+```
+
+Now that both of the central nodes **and** the *quorum device* server are sharing the same password, you will run this command **only on one of the central nodes** in order to authenticate on all the hosts taking part in the cluster.
```

Comment:
only on one of the cluster nodes, we don't care if it's on a central server nor the qdevice nor a database server